test/functionalTest/cases/0876_entity_dates/filter_datecreated_sub.test

```diff
@@ -26,7 +26,7 @@ Filter with dateCreated in subscriptions
 --SHELL-INIT--
 dbInit CB
 brokerStart CB
-accumulatorStart
+accumulatorStart --pretty-print
```

Comment:
Yeah, but what if we look at orionCurl instead?
The good way is to use 'flags', like orionCurl  (in my opinion).
I would suggest to (apart from implementing a real argv handler in the accumulator), try to use the flags in the orionCurl style. So much easier to use.  That's kind of why I decided to use --pretty-print and not prettyPrint